docs/content/components/borderbox.md

```diff
@@ -15,14 +15,26 @@ BorderBox is a Box component with a border.
 <iframe style="width: 100%; border: 0px; height: 350px;" srcdoc="<html><head><link href='https://unpkg.com/@primer/css/dist/primer.css' rel='stylesheet'></head><body><div class='Box '>    <div class='Box-header '>      Header</div>    <div class='Box-body '>      Body</div>    <ul>        <li class='Box-row '>          Row one</li>        <li class='Box-row '>          Row two</li>        <li class='Box-row '>          Row three</li>    </ul>    <div class='Box-footer '>      Footer</div></div></body></html>"></iframe>
 
 ```erb
-<%= render(Primer::BorderBoxComponent.new) do |component|
-  component.slot(:header) { "Header" }
-  component.slot(:body) { "Body" }
-  component.slot(:row) { "Row one" }
-  component.slot(:row) { "Row two" }
-  component.slot(:row) { "Row three" }
-  component.slot(:footer) { "Footer" }
-end %>
+<%= render(Primer::BorderBoxComponent.new) do |component| %>
+  <% component.slot(:header) do %>
+    Header
+  <% end %>
+  <% component.slot(:body) do %>
+    Body
+  <% end %>
+  <% component.slot(:row) do %>
+    Row one
+  <% end %>
+  <% component.slot(:row) do %>
+    Row two
+  <% end %>
+  <% component.slot(:row) do %>
+    Row three
```

Comment:
I think it might be useful to show that ERB is fully welcomed in this block. Something like:

```suggestion
    <% if true %> 
      <p>Row three</p>
     <% else %>
       <p>No Row</p>
    <% end %>
```